### PR TITLE
adding rspirep option for backend

### PR DESCRIPTION
**Pull request by floored1585**

### CODE REVIEW

**Comment by tersmitten** [comment]
Thanks for your contribution. Can you make sure it's added to all sections that support it?

https://cbonte.github.io/haproxy-dconv/1.8/configuration.html#rspirep

File: templates/etc/haproxy/backend.cfg.j2 [hunk 1]
```diff
@@ -57,6 +57,11 @@ backend {{ backend.name }}
   acl {{ acl.string }}
 {% endfor %}
 
+{% for rspirep in backend.rspirep | default([]) %}
+  rspirep {{ rspirep.string }}{% if rspirep.cond is defined %} {{ rspirep.cond }}{% endif %}
```
Comment (tersmitten) [inline templates/etc/haproxy/backend.cfg.j2]: Don't we need `<search> <string> <cond>`?

Comment (floored1585) [reply templates/etc/haproxy/backend.cfg.j2]: I will take a closer look soon, this solved my specific problem but will review again to generalize it

**Comment by floored1585** [comment]
I will update this soon with that in mind.

**Comment by tersmitten** [comment]
Thanks in advance

**Pull request merged**